mobile/test/common/integration/base_client_integration_test.h

```diff
@@ -74,6 +74,7 @@ class BaseClientIntegrationTest : public BaseIntegrationTest {
   Platform::StreamSharedPtr stream_;
   Platform::EngineSharedPtr engine_;
```

Comment:
Can we add a GUARDED_BY annotation to enforce that this must only be touched when the lock is held?

Reply:
yeah the only cross-thread issue was during start-up but I can lock all other access on principle.